src/renderer/gl_engine/meson.build

```diff
@@ -22,31 +24,6 @@ source_file = [
    'tvgGlTessellator.h',
 ]
 
-if host_machine.system() == 'darwin'
-    gl_dep = declare_dependency(link_args: ['-framework', 'OpenGL'])
-    compiler_flags += ['-Wno-deprecated']     #for deprecated opengl
-elif cc.get_id() == 'emscripten'
-    target_opengles = true
-    gl_dep = dependency('GLESv3', required: false)
-else
-    #find a opengl library with fallbacks
-    gl_dep = dependency('GL', required: false)
-
-    if not gl_dep.found()
-        gl_dep = meson.get_compiler('cpp').find_library('GL', required: false)
-    endif
-
-    if not gl_dep.found()
-        gl_dep = dependency('GLESv3', required: false)
-        target_opengles = true
-    endif
-
-    if not gl_dep.found()
-        gl_dep = meson.get_compiler('cpp').find_library('GLESv3')
-        target_opengles = true
-    endif
-endif
-
 if target_opengles
     gl_target_profile = '-DTHORVG_GL_TARGET_GLES=1'
```

Comment:
This looks like dead code. I think we can leave the build script above for confirming OpenGL/OpenGL ES.